chia/cmds/farm_funcs.py

```diff
@@ -219,13 +220,16 @@ class PlotStats:
     if all_harvesters is not None:
         harvesters_local: dict = {}
         harvesters_remote: dict = {}
+        harvesters_hostnames: dict = {}
         for harvester in all_harvesters["harvesters"]:
             ip = harvester["connection"]["host"]
             if is_localhost(ip):
                 harvesters_local[harvester["connection"]["node_id"]] = harvester
             else:
                 if ip not in harvesters_remote:
                     harvesters_remote[ip] = {}
+                    if socket.gethostbyaddr(ip):
+                        harvesters_hostnames[ip] = socket.gethostbyaddr(ip)[0]
```

Comment:
`gethostbyaddr` raises an exception if it fails, so we should rather do something like: 

```suggestion
                    try:
                        harvesters_hostnames[ip] = socket.gethostbyaddr(ip)[0]
                    except Exception:
                        pass
```